my-app/src/main/java/com/JavaWebServer/app/RESTMethods/Options.java

```diff
@@ -0,0 +1,12 @@
+package com.JavaWebServer.app;
+
+public class Options implements RestMethod {
+  private final String METHODKEY = "OPTIONS";
+  private final String ACCEPTED ="HTTP/1.1 200 ok";
```

Comment:
Also, Accepted may not be a good name for this, considering you are returning a 200 OK, rather than a 202 Accepted.